### PR TITLE
remote-builder support for gcloud alpha and beta commands.

### PR DESCRIPTION
**Pull request by stewartbryson**

### CODE REVIEW

**Comment by stewartbryson** [comment]
@googlebot I signed it!

**Comment by stewartbryson** [comment]
> Hi Stewart, thanks for your contribution! Could you please update the corresponding `README.md` with usage instructions or examples for the new functionality that you added? Your changes in `run-builder.sh` LGTM -- I'll run a `gcloud builds submit` smoke test and merge once the README is updated.

Yes... absolutely. Thank you.

File: remote-builder/run-builder.sh [hunk 1]
```diff
@@ -13,8 +13,9 @@ REMOTE_WORKSPACE=${REMOTE_WORKSPACE:-/home/${USERNAME}/workspace/}
 INSTANCE_NAME=${INSTANCE_NAME:-builder-$(cat /proc/sys/kernel/random/uuid)}
 ZONE=${ZONE:-us-central1-f}
 INSTANCE_ARGS=${INSTANCE_ARGS:---preemptible}
+GCLOUD=${GCLOUD:gcloud}
```
Comment (LOZORD) [inline remote-builder/run-builder.sh]: I think this needs to be `GCLOUD=${GCLOUD:-gcloud}` (missing hyphen) according to [this variable expansion guide](https://en.wikibooks.org/wiki/Bourne_Shell_Scripting/Variable_Expansion).

Comment (stewartbryson) [reply remote-builder/run-builder.sh]: I believe you are correct. I will fix now.l

**Pull request merged**